samples/tfm/tfm_psa_template/boards/nrf54l15dk_nrf54l15_cpuapp_ns.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_TFM_NRF_PROVISIONING=n
+CONFIG_TFM_DUMMY_PROVISIONING=y
```

Comment:
I should enable nrf_provisioning

Reply:
No, that can be out of scope for this PR.
This PR shows that the attestation report is generated and signed properly.
Provisioning can come later.
Do reviewers agree?

Reply:
I agree indeed, its good to limit the scope like this.